assets/training/vision/environments/automl-dnn-vision-gpu/context/conda_dependencies.yaml

```diff
@@ -27,4 +27,5 @@ dependencies:
   - azureml-interpret=={{latest-pypi-version}}
   - azureml-train-automl-runtime=={{latest-pypi-version}}
   - azureml-automl-dnn-vision=={{latest-pypi-version}}
-  - azureml-dataprep>=2.24.4
\ No newline at end of file
+  - azureml-dataprep>=2.24.4
+  - azure-identity>=1.16.1
```

Comment:
Could you please confirm if this is needed? I remember @nilesh wanted to try test without this package. Sorry, I'm not aware if he has done it.

Reply:
We can do it in the next iteration.